readme.txt

```diff
@@ -4,7 +4,7 @@ Tags: course progress, sensei lms, widget
 Requires at least: 4.9
 Tested up to: 5.3
```

Comment:
Should we update it here too?

Reply:
Oops! https://github.com/woocommerce/sensei-course-progress/pull/101/commits/2679e5652122b95ebe841c502fed9011e4403826